includes/class-newspack-listings-core.php

```diff
@@ -841,7 +841,9 @@ public static function enable_listing_category_archives( $query ) {
 		}
 
 		if ( ! is_admin() && $query->is_main_query() ) {
-			if ( is_category() || is_tag() ) {
+			$archive_should_include_listings = is_category() || is_tag();
```

Comment:
This means that a listing buyer can assign their listing to all site's categories, and then that listing will appear among other listings on category pages. Is that intended?

Reply:
For now, yes, because listings must be approved by an editor or admin user before being published (so they would have an opportunity to remove unwanted categories and tags). In a future update we could perhaps limit the assignment to certain UGC-specific terms only.